client/src/app/tabs/getEditMenu.js

```diff
@@ -254,6 +256,16 @@ export function getSelectionEntries({
     });
   }
 
+
+  if (isDefined(replaceElement)) {
+    menuEntries.push({
+      label: 'Replace element',

```

Comment:
I would suggest the label "Replace Selected" instead, so it is aligned with the other entry ("Remove Selected")

![image](https://user-images.githubusercontent.com/21984219/210248655-b1426f04-825b-4716-912e-c09a31a4cdcb.png)

```suggestion
      label: 'Replace Selected',
```

Reply:
That's also fine with me. We should still use title case for the labels:
```suggestion
      label: 'Replace Element',
```